tests/test_azure.py

```
@@ -0,0 +1,160 @@
+def azurite():
+    """Ensure Azurite is running. Try in order: already running (CI), start as
+    subprocess, skip."""
+    if _azurite_running():
+        yield None  # Already running (e.g. started by CI step).
+        return
+
+    azurite_bin = shutil.which("azurite-blob")
+    if not azurite_bin:
+        pytest.skip("azurite-blob not found and Azurite is not already running")
+
+    proc = subprocess.Popen(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)